manifest.yaml

```diff
@@ -1,5 +1,6 @@
 name: github.com/getoutreach/stencil-golang
-## <<Stencil::Block(keys)>>
+###Block(keys)
```

Comment:
Looks like old style comments ended up here somehow.

Reply:
yeah, i need to update it in `stencil-template-base`, going to just keep it for now